deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -44,7 +56,8 @@ def callback_detect_vulnerability_scan_sleeping(line):
 
 
 def callback_detect_update_from_year_result(line):
-    invalid_year_msg = r'.*wazuh-modulesd.*wmodules-vuln-detector.*ERROR: Invalid content for \'update_from_year\' option at module \'vulnerability-detector\''
+    invalid_year_msg = r'.*wazuh-modulesd.*wmodules-vuln-detector.*ERROR: Invalid content for \'update_from_year\'\
+                         option at module \'vulnerability-detector\''
```

Comment:
```suggestion
    invalid_year_msg = r".*wazuh-modulesd.*wmodules-vuln-detector.*ERROR: Invalid content for 'update_from_year'\
                         option at module 'vulnerability-detector'"
```

Reply:
Done in 77223ed24d8fff611947e817d180bf744720ed4c